documentation/src/main/asciidoc/topics/con_performance_clustered_cache_modes.adoc

```diff
@@ -0,0 +1,139 @@
+To take the most advantage of this elasticity, you should consider how scaling the number of nodes up or down affects cache capacity.
+
+For replicated caches, each time a node joins the cluster it receives a complete copy of the data set.
+This means it takes more time for nodes to join but also imposes a limit on overall capacity because a replicated cache can never exceed the amount of memory available to the host.
+For example, if the size of your data set is 10Gb then each node must have at least 10Gb of available memory.
+
+On the other hand, adding more nodes increases capacity for a distributed cache as each member of the cluster stores only a subset of the data.
+To store 10Gb of data you can have eight nodes with 2.5Gb of available memory each if the number of owners is two, without taking memory overhead into consideration.
+Each additional node that joins the cluster increases the capacity of the distributed cache by 2.5Gb.
+
+In this way, the capacity of a distributed cache is not bound by the amount of memory available to underlying hosts.
+
+[discrete]
+== Synchronous or asynchronous replication
+
+{brandname} can communicate synchronously or asynchronously when primary owners send replication requests to backup nodes.
+
+[%autowidth,cols="1,1",stripes=even]
+|===
+|Replication mode | Effect on performance
+
+|**Synchronous**
+|Synchronous replication helps to keep your data consistent but adds latency to cluster traffic that reduces throughput for cache writes.
+
+|**Asynchronous**
+|Asynchronous replication removes latency and increases the speed of write operations but leads to data inconsistency and provides a lower guarantee against data loss.
+|===
+
+With synchronous replication, {brandname} notifies the originating node when replication requests complete on backup nodes.
+{brandname} retries the operation if a replication request fails due to a change to the cluster topology.
+When replication requests fail due to other errors, {brandname} throws exceptions for client applications.
+
+With asynchronous replication, {brandname} does not provide any notification for replication requests.
+This has the same effect for applications as all requests being successful.
+On the {brandname} cluster, however, the primary owner has the correct entry but one or more of the backup nodes either does not have the entry or has an out of date copy of the entry.
+In the event that the primary owner crashes, you potentially lose data.
+
+To prevent data loss with asynchronous replication, the primary owner adds failed requests to a queue and uses a background thread to retry the operations.
+In environments with high degrees of cluster instability, where it is not possible for nodes to rebalance, the queue of failed replication requests can increase in size and lead to out of memory issues for primary owners.
+However, under normal cluster conditions, any subsequent write operations on the same key remove it from the queue.
```

Comment:
@danberindei Could you please review this section again? I made some edits based on feedback from Wolf and I wonder if I'm losing nuances or changing meaning. Thanks.